minimal.slim

```diff
@@ -96,7 +89,7 @@ late() {
 	}
 }
 
-s1 late() {
+RUNTIME late() {	// BCH: rescheduleScriptBlock() no longer needed, yay!
```

Comment:
yesssss

Reply:
Removed all reschedule blocks! It's very satisfying.